File: src/main/java/github/zmilla93/core/chatparser/IChatScannerListener.java
package github.zmilla93.core.chatparser;

import github.zmilla93.core.data.PlayerMessage;
import github.zmilla93.gui.chatscanner.ChatScannerEntry;

public interface IChatScannerListener {

    void onScannerMessage(ChatScannerEntry entry, PlayerMessage message, boolean loaded);

}

File: src/main/java/github/zmilla93/core/chatparser/IChatScannerListener.kt
package github.zmilla93.core.chatparser

import github.zmilla93.core.data.PlayerMessage
import github.zmilla93.gui.chatscanner.ChatScannerEntry

interface IChatScannerListener {
    fun onScannerMessage(entry: ChatScannerEntry?, message: PlayerMessage?, loaded: Boolean)
}
